packages/mui-envinfo/envinfo.js

```diff
@@ -7,6 +7,7 @@ envinfo
     {
       npmPackages: `{${[
         '@mui/*',
+        '@toolpad/*',
         // Peer dependencies
```

Comment:
we are missing `pigment-css/*` and `@base_ui/*` as far as I can tell.

Reply:
Opening https://github.com/mui/material-ui/pull/43002